go.mod

```diff
@@ -71,6 +70,7 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	k8s.io/apiextensions-apiserver v0.26.1 // indirect
 	k8s.io/component-base v0.26.1 // indirect
+	k8s.io/klog/v2 v2.80.1 // indirect
```

Comment:
I was dry running the build process and goreleaser had me run `go mod tidy`. LMK if I should undo the bits here.